docs/CONTRIBUTING.md

```diff
@@ -102,6 +102,49 @@ This usually involved editing the `Containerfile`. Most techniques for building
 
 Check out CoreOS's [layering examples](https://github.com/coreos/layering-examples) for more information on customizing.
 
+## Using Blue-build locally
+### Requirements
+
+- [Blue-build CLI](https://github.com/blue-build/cli)
+- Podman
+- git
+
+Secureblue already includes `bluebuild` but running locally requires customizing policy.json for your user to allow pulling a few unsigned images.
+
+#### Policy.json configuration
+
+On secureblue only pre-configured signed images are allowed to be pulled. Four repos need to be configured:
+
+- docker.io/mikefarah/yq `Unsigned`
+- gcr.io/projectsigstore/cosign `Signed`
+- ghcr.io/blue-build/cli `Unsigned`
+- ghcr.io/blue-build/modules `Unsigned`
+
+Add rules using podman CLI:
+
+- `podman image trust set --type accept docker.io/mikefarah/yq`
+- `podman image trust set --type accept gcr.io/projectsigstore/cosign`
+- `podman image trust set --type accept ghcr.io/blue-build`
+
+TODO: Optionally enable sigstore checks for `gcr.io/projectsigstore/cosign` by adding to your `~/.config/containers/policy.json`
```

Comment:
if it's signed, let's add the pubkey and config to the default policy :smile:

Reply:
I checked the pubkey through `cosign tree` but it seems their gcr.io and ghcr.io images are signed differently. I'll look for it tomorrow if it can be retrieved from the image or repo somehow.

Reply:
Added config for cosign to default policy.